src/Broadway/EventStore/EventStoreInterface.php

```diff
@@ -28,6 +28,8 @@ public function load($id);
     /**
      * @param mixed                      $id
      * @param DomainEventStreamInterface $eventStream
+     *
+     * @throws EventStoreException
```

Comment:
I think it's better to specify concrete exceptions, i.e. `@throws DuplicatePlayheadException`.